test/fixtures/fixture-app.js

```diff
@@ -0,0 +1,8 @@
+var proxyquire = require('proxyquire');
+var mocks = {
+  'sharepointer' : require('./fixture-sharepointer'),
+  'fh-mbaas-api' : require('./fixture-mbaas-api')
+};
+mocks.sharepointer['@global'] = true;
+mocks['fh-mbaas-api']['@global'] = true;
+module.exports = proxyquire('../../application.js', mocks);
```

Comment:
this makes sense as a reasonable way to use a mock sharepointer for functional testing the routes